### DISCUSSION TITLE
Device modelID correct but "unsupported" - Samotech SM323

### DISCUSSION THREAD

**Discussion by alexyoung97** [Koenkk/zigbee2mqtt#16340] [Devices]
Hi,

I just connected up my [Samotech SM323](https://www.zigbee2mqtt.io/devices/SM323.html) and whilst zigbee2mqtt can see it, it shows it as an unsupported device named "HK_DIM_A". This is despite zigbee2mqtt supporting this device (I can see it in the Samotech [converters](https://github.com/Koenkk/zigbee-herdsman-converters/blob/master/devices/samotech.js#L65) file).

My database.db file contains this:

```
{"id":19,"type":"Router","ieeeAddr":"0x84b4dbfffe089ce6","nwkAddr":38077,"manufId":4107,"manufName":"Samotech","modelId":"HK_DIM_A","epList":[1,242],"endpoints":{"1":{"profId":260,"epId":1,"devId":257,"inClusterList":[0,3,4,5,6,8,2821,4096],"outClusterList":[25],"clusters":{"genBasic":{"attributes":{"appVersion":0,"stackVersion":0,"hwVersion":0}}},"binds":[],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":0,"stackVersion":0,"hwVersion":0,"dateCode":"20220531","swBuildId":"1.00","zclVersion":8,"interviewCompleted":true,"meta":{},"lastSeen":1674591086610,"defaultSendRequestWhen":"immediate"}
```

Which shows `manufName":"Samotech","modelId":"HK_DIM_A"`. So I don't know why it is not being recognised.

Could anyone help me to understand this? Thanks!

**Comment by Koenkk** [comment]
The `HK_DIM_A` was recently added and will be supported out-of-the-box with the 1 February release.

**Reply by alexyoung97** [reply]
Ah, thanks - I should have checked the current release tag rather than master, doh!
